scripts/update_sql.sh

```diff
@@ -2,13 +2,15 @@
 
 cd $(dirname $0)/../mbdata
 
+TARGET_BRANCH="${1:-master}"
+
 rm -rf sql
 
 rm -rf /tmp/mbserver-clone
 mkdir /tmp/mbserver-clone
 
-curl -L -o /tmp/mbserver-clone/master.tar.gz https://github.com/metabrainz/musicbrainz-server/archive/master.tar.gz
-tar -x -f /tmp/mbserver-clone/master.tar.gz --strip-components=2 --wildcards 'musicbrainz-server-master/admin/sql/*'
+curl -L -o /tmp/mbserver-clone/$TARGET_BRANCH.tar.gz https://github.com/metabrainz/musicbrainz-server/archive/$TARGET_BRANCH.tar.gz
+tar -x -f /tmp/mbserver-clone/$TARGET_BRANCH.tar.gz --strip-components=2 --wildcards "musicbrainz-server-$TARGET_BRANCH/admin/sql/*"
```

Comment:
* Follow the above suggestion
* Quote variable references

```suggestion
curl -L -o "/tmp/mbserver-clone/$GIT_TARGET.tar.gz" "https://github.com/metabrainz/musicbrainz-server/archive/$GIT_TARGET.tar.gz"
tar -x -f "/tmp/mbserver-clone/$GIT_TARGET.tar.gz" --strip-components=2 --wildcards "musicbrainz-server-$TARGET_BRANCH/admin/sql/*"
```